### PR TITLE
hmset can be called without function callback

### PR DESCRIPTION
**Pull request by liorsion**
sometimes I just want to use an array to hmset without the need for callback

### CODE REVIEW

**Comment by liorsion** [comment]
@DTrejo commenting with indignation..

File: index.js [hunk 1]
```diff
@@ -1046,7 +1046,7 @@ RedisClient.prototype.HMGET = RedisClient.prototype.hmget;
 RedisClient.prototype.hmset = function (args, callback) {
     var tmp_args, tmp_keys, i, il, key;
 
-    if (Array.isArray(args) && typeof callback === "function") {
+    if (Array.isArray(args) && (typeof callback === "function" || callback === undefined)) {
```
Comment (BridgeAR) [inline index.js]: Removing the check for callback would be better.

**Comment by BridgeAR** [comment]
@liorsion if you remove the check and add a test, this should be fine to merge.

**Comment by BridgeAR** [comment]
Reference #321

**Pull request closed without merging**